pkg/provider/remote/binary.go

```diff
@@ -0,0 +1,128 @@
+// Copyright Nitric Pty Ltd.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package remote
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+
+	"github.com/pterm/pterm"
+
+	"github.com/nitrictech/cli/pkg/output"
+	"github.com/nitrictech/cli/pkg/provider/types"
+	"github.com/nitrictech/cli/pkg/utils"
+)
+
+// Provides a binary remote provider type
+type binaryRemoteDeployment struct {
+	providerPath    string
+	autoPulumiLogin bool
+	*remoteDeployment
+}
+
+func (p *binaryRemoteDeployment) startProcess() (*os.Process, error) {
+	cmd := exec.Command(p.providerPath)
+
+	if p.autoPulumiLogin {
+		// Retrieve the current environment variables
+		env := os.Environ()
+		cmd.Env = append(env, "PULUMI_BACKEND_URL=file://~")
+	}
```

Comment:
remove this, logic for appending environment variables can be put in through the nitric provider.